Documentation/helm-operator.md

```diff
@@ -148,6 +154,9 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.volumeReplication.image`       | Volume Replication Controller image.                                                                                        | `quay.io/csiaddons/volumereplication-operator:v0.3.0`     |
 | `csi.csiAddons.enabled`     | Enable CSIAddons                                                                                                  | `false`                                                   |
 | `csi.csiAddons.image`       | CSIAddons Sidecar image.                                                                                        | `quay.io/csiaddons/k8s-sidecar:v0.2.1`     |
+| `csi.nfs.enabled`     | Enable nfs driver.                                                                                                  | `false`                                                   |
+| `csi.nfs.image`       | NFS nodeplugin image. image.                                                                                        | `mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0`     |
+| `csi.nfs.probe.image`       | NFS nodeplugin liveness probe image. image.                                                                                        | `k8s.gcr.io/sig-storage/livenessprobe:v2.5.0`     |
```

Comment:
Helm best practices suggest not nesting values. We already have the `.csi` nest, and we already have the `.image` nest format, but I think we can still eliminate the `.nfs` nest. I think this would be better as `csi.nfsProbe.image`.

Reply:
removed probe, 
I think having nfs -> enabled + image is cleaner approach.
and similar to volrep and csiaddons.